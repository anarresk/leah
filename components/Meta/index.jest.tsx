import React from 'react';

import {Meta} from './index';
import expectShallow from '../../utils/testing/expectComponent';

describe('components/Meta', () => {
    it('renders as expected', () => {
        expectShallow(<Meta />).toMatchSnapshot();
    });
});
