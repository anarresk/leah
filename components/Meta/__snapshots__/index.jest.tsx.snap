// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Meta renders as expected 1`] = `
<SideEffect(Head)>
  <meta
    content="width=device-width, initial-scale=1"
    name="viewport"
  />
  <meta
    charSet="utf-8"
  />
</SideEffect(Head)>
`;
